### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by KCraw**
screen shot 2016-02-16 at 3 52 51 pm
screen shot 2016-02-16 at 4 05 11 pm

I redid everything paying attention to flexbox and RN guidelines for what styles are permissible for what native view types. 

Components should now be reusable/customizable and there shouldn't be any fixed sizes anywhere.

Everything a little closer to Material.

### CODE REVIEW

File: components/app/styles.js [hunk 1]
```diff
@@ -2,13 +2,10 @@
...
-      width: dimensions.width,
-      height: dimensions.height - 20,
-      backgroundColor: '#cccccc',
-      marginTop: 20,
-    },
-  });
-}
+export default StyleSheet.create({
+  scene: {
+    flex: 1,
```
Comment (danmconrad) [inline components/app/styles.js]: way better

File: components/button/index.js [hunk 2]
```diff
@@ -10,10 +10,17 @@ import React, {
 import styles from './styles';
 
 export default class Button extends Component {
+
+  static propTypes = {
+    buttonStyle: View.propTypes.style,
+    buttonTextStyle: Text.propTypes.style,
+    onPress: React.PropTypes.func,
```
Comment (danmconrad) [inline components/button/index.js]: nice

File: components/recommendation-scene/index.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ export default class RecommendationScene extends Component {
       <View style={[this.props.style, styles.scene]}>
         <RecommendationComponent rec={rec} onToggleDetail={this.toggleExpanded.bind(this)} />
 
-        <Button style={styles.commitButton} onPress={this.commitRec.bind(this)}>
+        <Button buttonStyle={styles.commitButton} onPress={this.commitRec.bind(this)}>
```
Comment (danmconrad) [inline components/recommendation-scene/index.js]: very cool.

**Comment by danmconrad** [comment]
:shipit: (can you attach a screen cap?)

**Comment by KCraw** [comment]
screen shot 2016-02-16 at 3 52 51 pm


screen shot 2016-02-16 at 4 05 11 pm

**Comment by jimbol** [comment]
I'm mergin

**Pull request merged**